subtests/docker_cli/run_cgroups/memory.py

```diff
@@ -185,7 +189,11 @@ def check_result(self, subargs, long_id):
         return self.check_memory(memory_value, cgroup_memory, memory_unit)
 
     def postprocess_positive(self, this_result, passed):
-        self.failif(not isinstance(this_result, dict))
+        if this_result is None:
+            raise xceptions.DockerTestError("Invoked with null results")
+        self.failif(not isinstance(this_result, dict),
+                    ("expected this_result to be a dict; it's a %s" %
+                     this_result.__class__))
```

Comment:
Nice!  Yeah, inconsistent typing like this is soooo confusing when reading a test.